src/EventService.Data/EventFileRepository.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Data.Provider;
+using LT.DigitalOffice.EventService.Models.Db;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.File;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.EventService.Data;
+
+public class EventFileRepository : IEventFileRepository
+{
+  private readonly IDataProvider _provider;
+
+  public EventFileRepository(
+    IDataProvider provider)
+  {
+    _provider = provider;
+  }
+
+  public async Task<List<Guid>> CreateAsync(List<DbEventFile> files)
+  {
+    if (files is null)

```

Comment:
Will empty list be handled? mb add & !files.Any()?

Reply:
done